README.md

```diff
@@ -1 +1,122 @@
-# nested-hash-iteration
+# Iterating Over Nested Hashes
+
+##Objectives
+* Understand how to iterate over a nested hash
+* Understand where you are at each level of the iteration
+
+-
+
+At this point you should be familiar with iterating over hashes that have one level—a series of key/value pairs on a single tier. 
+
+```ruby
+jon_snow = {
+  name: "Jon",
+  email: "jon_snow@thewall.we"
+}
+```
+
+What happens when we want to iterate over a multidimensional hash like the one below? Let's iterate over our nested hash one level at a time; iterating over the first level of our hash would look like this: 
+
+```ruby
+contacts = {
+  "Jon Snow" => {
+    name: "Jon",
+    email: "jon_snow@thewall.we", 
+    favorite_icecream_flavors: ["chocolate", "vanilla", "mint chip"],
+    knows: nil
```

Comment:
Can you drop the `knows: nil` stuff? That is definitely not a great thing to show them. Mostly because of this:

```
h = {test: 1, test2: 3, test3: nil}

h[:test1] #returns 1 Awesome
h[:test3] #returns nil
h[:blahblahblah] #returns nil.
```
Notice how by setting something to nil explicitly, you are unable to actually see if you set it to nil, or if the key just doesn't exist? bad